rest/data/project.go

```diff
@@ -161,7 +161,11 @@ func (pc *DBProjectConnector) FindProjectVarsById(id string) (*restModel.APIProj
 			Message:    fmt.Sprintf("variables for project '%s' not found", id),
 		}
 	}
-	vars.RedactPrivateVars()
+	if redactedOnly {
+		vars.RedactedOnly()
+	} else { // only return values for redacted variables
+		vars.RedactPrivateVars()
```

Comment:
would it be better if these had return values?

Reply:
I was going to say no but I had a legitimate dream where a cartoon character yelled at me for using destructively modifying functions sooo I guess yes